src/plugin.js

```diff
@@ -9,6 +9,7 @@ import redispatch from './redispatch.js';
 import initializeContentupdate from './contentupdate.js';
 import adMacroReplacement from './macros.js';
 import cueTextTracks from './cueTextTracks.js';
+import pm from './playMiddleware.js';
```

Comment:
I suggest spelling it out as `import playMiddleware from './playMiddleware.js';` to follow the project convention and make the code more readable.